### ISSUE TITLE
Ground-truth of validation dataset  may have leaked when training

### ISSUE THREAD

**Issue by Fangkang515** [zyqz97/GP-NeRF#6]
Thanks for your work.

When using Mega-NeRF and GP-NeRF, I have noticed that the validation set might have been inadvertently included in the training set (in this [issue](https://github.com/cmusatyalab/mega-nerf/issues/67)). It seems that this issue has also been carried over to GP-NeRF.

I'm not certain if I may have overlooked an important part of the code, but if there are any errors, please feel free to point out my mistakes.

**Comment by zyqz97** [comment]
Refer to [this issue](https://github.com/cmusatyalab/mega-nerf/issues/18)
